plugins/sip/openipv1/endpoint/body.json

```diff
@@ -0,0 +1,32 @@
+{
+    "label": "{{ label }}",
+    "name": "openip_trunk_{{ tenant_uuid }}_{{ suffix }}",
+    "transport": {"uuid": "{{ transport_uuid }}"},
+    "templates": ["{{ global_sip_template_uuid }}", "{{ registration_trunk_sip_template_uuid }}"],
+    "endpoint_section_options": [
+        ["identify_by", "header,auth_username,username"],
+        ["allow", "!all,alaw"]
+    ],
+    "registration_section_options": [
+        ["contact_user", "{{ username }}"],
+        ["client_uri", "sip:{{ username }}@voip.myopenip.fr:5060"],
+        ["expiration", "60"],
+        ["line", "yes"],
+        ["server_uri", "sip:voip.myopenip.fr:5060"]
```

Comment:
Is it required to add `:5060`? I know that the UI may remove it because it's the default port.

Reply:
If we use 5070, how do it works?

Reply:
Don't know Sylvain if your question is for me or Pascal. I'm just asking if we should add port only when it's not `5060`, like we do in Portal right now. So if we have 5070, we can add it in the `server_uri`, `client_uri`, etc.